examples/CMakeLists.txt

```diff
@@ -1,15 +1,52 @@
-include_directories(${PROJECT_SOURCE_DIR}/include)
-include_directories(${PROJECT_SOURCE_DIR}/plugins)
-include_directories(${PROJECT_BINARY_DIR})
-include_directories(${PROJECT_SOURCE_DIR}/examples)
-include_directories(${PROJECT_SOURCE_DIR}/arch)
-include_directories(${PROJECT_SOURCE_DIR}/plugins/historydata)
-
-set(examples_headers
-    ${examples_headers}
-    ${PROJECT_SOURCE_DIR}/examples/common.h
-    )
-
+if(UA_EXAMPLES_STANDALONE)
+    #
+    # test Debian packaging resp. build only examples
+    #
+    # cp -a /usr/share/open62541/examples .
+    # OR
+    # cp -a <path-to-open62541-src-tree/examples .
+    # cd examples
+    # mkdir build
+    # cd build
+    # cmake -DUA_EXAMPLES_STANDALONE=ON ..
+    # OR
+    # cmake -DUA_EXAMPLES_STANDALONE=ON -DUA_NAMESPACE_ZERO=FULL ..
+    # make
+    cmake_minimum_required(VERSION 3.0)
+    project(open62541)
+    find_package(open62541 0.4.0 REQUIRED)
```

Comment:
Can you remove the specific version number. Otherwise we always need to update this if we have a version change.